website/docs/docs/collaborate/data-health-signals.md

```diff
@@ -0,0 +1,87 @@
+| 🔴 **Degraded**  | - Model failed to build OR<br />- Model has failing tests OR<br />- One or more upstream sources are stale:<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check hasn’t run in the past 30 days<br />&nbsp;&nbsp;&nbsp;&nbsp;- Freshness check returned an error |
+| ⚪ **Unknown**    | - Unable to determine health of resource; no job runs have processed the resource.         |
+
+</TabItem>
+
+<TabItem value="sources" label="Sources">
+
+The health state of a source is determined by the following criteria:
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ Healthy	| - Freshness check configured AND<br />- Freshness check passed AND<br />- Freshness check ran in the past 30 days AND<br />- Has a description |
+| 🟡 Caution	| - Freshness check returned a warning OR<br />- Freshness check not configured OR<br />- Freshness check not run in the past 30 days OR<br />- Missing a description |
+| 🔴 Degraded	| - Freshness check returned an error |
+| ⚪ Unknown	| Unable to determine health of resource; no job runs have processed the resource.     |
+
+</TabItem>
+
+<TabItem value="exposures" label="Exposures">
+
+The health state of an exposure is determined by the following criteria:
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ Healthy	| - Underlying sources are fresh AND<br />- Underlying models built successfully AND<br />- Underlying models’ tests passing AND<br />- (TBD) Underlying models built in the last 30 days |
+| 🟡 Caution	| - At least one underlying source’s freshness checks returned a warning OR<br />- At least one underlying model was skipped OR<br />- At least one underlying model’s tests returned a warning OR<br />- (TBD) At least one model not built in the last 30 days |
+| 🔴 Degraded	| At least one underlying source’s freshness checks returned an error OR<br />- At least one underlying model did not build successfully OR<br />- At least one model’s tests returned an error |
+
+</TabItem>
+
+<TabItem value="source-collection" label="Source collection health">
+
+The health state of a source collection is determined by the following criteria:
+
+Functions as an aggregate of underlying sources
+
+| **Health state** | **Criteria**   |
+|-------------------|---------------|
+| ✅ Healthy	| - All underlying sources have freshness checks configured OR<br />- All passed their freshness checks OR<br />- All freshness checks ran in the past 30 days OR<br /> - All sources have a description |
+| 🟡 Caution	| - One or more sources lack freshness checks OR<br />- One or more freshness checks returned a warning OR<br />- One or more freshness checks not run in the past 30 days OR<br />- One or more sources missing a description |
+| 🔴 Degraded	| - One or more underlying sources’ freshness checks returned error |
+
+</TabItem>
```

Comment:
Could you remove source collections? These can be added in if/when we implement META-3973/3971